javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const width = 5;
+const height = 4;
+const area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+const bill = 15;
+const payment = 20;
+const change = bill - payment;
+console.log('value of change:', change);
+console.log('typeof change', typeof change);
+const quizzes = 6;
+const midterm = 11;
+const final = 7;
+const grade = (quizzes + midterm + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+const firstName = `Andrew `;
+const lastName = `Ortiz`;
+const fullName = `${firstName}${lastName}`;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+const ph = 10;
+const isAcidic = ph < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+const headCount = 200;
+const isSparta = headCount === 300;
+console.log('value of isSparta:', headCount === isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+let motto = `${firstName}${lastName}`;
```

Comment:
This still needs to be updated. You should initialize `motto` to a value of `fullName`, but then using the `+=` operator you add a string, like what you're doing on line 31. You're more than welcome to use template literals in the future, but for this exercise, you need to do the `+=`